src/ucp/rndv/rndv.c

```diff
@@ -1769,11 +1784,17 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_atp_handler,
                  (arg, data, length, flags),
                  void *arg, void *data, size_t length, unsigned flags)
 {
+    ucp_worker_h worker      = arg;
     ucp_reply_hdr_t *rep_hdr = data;
-    ucp_request_t *req;
+    ucp_request_t *rtr_sreq, *req;
+
+    UCP_SEND_REQUEST_GET_BY_ID(&rtr_sreq, worker, rep_hdr->req_id, 1,
+                               return UCS_OK, "RNDV ATP %p", rep_hdr);
 
-    UCP_WORKER_GET_REQUEST_BY_ID(&req, arg, rep_hdr->req_id, return UCS_OK,
-                                 "RNDV ATP %p", rep_hdr);
+    req = rtr_sreq->super_req;
```

Comment:
req -> rreq

Reply:
this was not named as `rreq`, because it is used for FRAG protocol as a request for staging

Reply:
is it "freq" then?

Reply:
it either freq or rreq